lib/manageiq_performance/reporter.rb

```diff
@@ -143,7 +143,7 @@ def column_size_for header, request_id
         (HEADERS[header].map { |header_column|
           @report_data[request_id][header_column].map {|i|
             value = i.to_s
-            value = "%.1f" % value unless value.class.ancestors.include?(Integer)
+            value = "%.1f" % i if i && !i.class.ancestors.include?(Integer)
```

Comment:
This isn't really your fault, but I am seeing this same code in line 127 and wondering why it isn't an issue.

Turns out, it is because the block above it is converting everything to an int in it for some reasons, which doesn't make sense if we ever want a float value to be displayed in the report (maybe we don't...).